pages/content/reports/2022/index.md

```diff
@@ -0,0 +1,391 @@
+
+- OWD was the single organization with the greatest number of PRs merged to the MDN content repository.
+- OWD provided 60% of all pull request reviews to the MDN content repository.
+- The OWD team executed a number of writing and documentation infrastructure projects, including:
+  - Modernizing the JavaScript used in thousands of code samples across MDN, ensuring all examples demonstrate modern best practices
+  - Creating and extending documentation to meet the needs identified in the [Interop 2022](https://web.dev/interop-2022/) project, including documenting CSS Colors and Cascade layers,
+  - Completely rewriting the documentation for the Performance APIs
+  - Rewriting CSS core principles of inheritance, specificity and the cascade.
+  - Documenting all ARIA roles and attributes,
+  - Making MDN more inclusive by ensuring all examples identify document language, have an appropriate character set, and are responsive, every image has alt text, and human names used in code examples better reflect the diversity of those reading MDN.
+  - Establishing [overall themes](https://github.com/openwebdocs/project/blob/main/steering-committee/themes.md) for prioritized work
+
+## What is MDN and why does it matter?
+
+In 2022, OWD staff primarily worked on improving and extending [MDN Web Docs](https://developer.mozilla.org).
+
+MDN is a website documenting front-end open web technologies, in particular JavaScript, CSS, HTML, HTTP, ARIA, SVG, MathML, and the JavaScript APIs available in web browsers.
+
+MDN is the top destination for developers to learn about web technologies. It's not just a popular website for web developers, it is an essential tool that they consult on a daily basis in their work. It provides value to all organizations that depend on the web to any extent .
+
+- MDN is the second most cited documentation site on Stack Overflow, after learn.microsoft.com ([Source](https://meta.stackoverflow.com/questions/417189/which-sites-are-currently-linked-to-most-often-in-stack-overflow/417190#417190))
+- MDN is visited by 17 million developers every month
+- MDN's JavaScript, CSS, and HTML documentation are routinely seen as the standard language references
+- MDN is the only site with comprehensive reference documentation for the web platform APIs
+
+## MDN content maintenance
+
+Infrastructure and operational costs for operating and maintaining the MDN website at developer.mozilla.org are borne by Mozilla, but the largest and most valuable part of MDN is the content itself. The web platform is big and constantly evolving: MDN contains over 11,000 pages, and the content needs ongoing updating and maintenance as new web platform features are introduced and best practices change.
+
+MDN's content is maintained as an open source project with active contributors including Mozilla staff, Open Web Docs staff, various individuals paid by their employer to contribute, and a volunteer community.
```

Comment:
The way that I tried rephrasing the previous paragraph might make this superfluous.